frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -1,20 +1,46 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
 import styles from './styles.module.scss';
 
 type Properties = {
   label: string;
+  checked?: boolean;
+  name?: string;
   onChange: () => void;
+  disableDefaultStyles?: boolean;
 };
 
-const Checkbox: React.FC<Properties> = ({ label, onChange }) => {
+const Checkbox: React.FC<Properties> = ({
+  label,
+  checked,
+  name,
+  onChange,
+  disableDefaultStyles,
+}) => {
+  const checkboxClassName = getValidClassNames(
+    disableDefaultStyles ? null : styles['checkbox'],
+  );
+  const containerClassName = getValidClassNames(
+    disableDefaultStyles ? null : styles['container'],
+  );
+  const labelClassName = getValidClassNames(
+    disableDefaultStyles ? null : styles['label'],
+  );
+  const checkboxContainerClassName = getValidClassNames(
+    disableDefaultStyles ? styles['checkbox-container'] : null,
+  );
+
   return (
-    <label>
+    <label className={checkboxContainerClassName}>
       <input
-        className={styles['checkbox']}
+        className={checkboxClassName}
         type="checkbox"
+        name={name}
```

Comment:
```suggestion
```
You don't use control and just overwrite field onChange, so you shouldn't pass name at all